proto/babylon/epoching/v1/query.proto

```diff
@@ -224,6 +225,8 @@ message QueuedMessageResponse {
   // msg is the actual message that is sent by a user and is queued by the
   // epoching module as string.
   string msg = 5;
+  // enriched_msg contains a parsed, structured version of the original message
+  string queued_msg_type = 6;
```

Comment:
Docstring is not consistent with field. Also maybe we can simply call it `msg_type`